README.md

```diff
@@ -98,10 +98,10 @@ This plugin tries to add SonarQube issues to your project configuration files (e
 To configure the severity of the created issues you can optionally specify the minimum score for each severity with the following parameter. Specify a score of `-1` to completely disable a severity.
 
 ```ini
-sonar.dependencyCheck.severity.blocker=9.0
-sonar.dependencyCheck.severity.critical=7.0
-sonar.dependencyCheck.severity.major=4.0
-sonar.dependencyCheck.severity.minor=0.0
+sonar.dependencyCheck.severity.critical=9.0
```

Comment:
This refers to the SonarQube severity level. Critical should not be here.

Reply:
Agreed. It was left in until the CVSS scores were disentangled from the SonarQube scores.